examples/textcat_binary.yaml

```diff
@@ -0,0 +1,9 @@
+- text: "This is a recipe for scrambled egg: 2 eggs, 1 butter, batter them, and then fry in a hot pan for 2 minutes"
```

Comment:
Final readme comment: the document should mention that you can add context with these examples. I've really seen it improve the output so it's good to point folks' attention to it.